File: documentation/user/en/query/requirements/examples/fetching/referenceContentWithAttributes.java
final EvitaResponse<SealedEntity> entities = evita.queryCatalog(
	"evita",
	session -> {
		return session.querySealedEntity(
			query(
				collection("Product"),
				filterBy(
					entityPrimaryKeyInSet(105703)
				),
				require(
					entityFetch(
						attributeContent("code"),
						referenceContentWithAttributes(
							"parameterValues",
							attributeContent("variant"),
							entityFetch(
								attributeContent("code")
							),
							entityGroupFetch(
								attributeContent("code")
							)
						)
					)
				)
			)
		);
	}
);

File: documentation/user/en/query/requirements/examples/fetching/referenceContentWithAttributes.cs
EvitaResponse<ISealedEntity> entities = evita.QueryCatalog(
	"evita",
	session => session.QuerySealedEntity(
        Query(
        	Collection("Product"),
        	FilterBy(
        		EntityPrimaryKeyInSet(105703)
        	),
        	Require(
        		EntityFetch(
        			AttributeContent("code"),
        			ReferenceContentWithAttributes(
        				"parameterValues",
        				AttributeContent("variant"),
        				EntityFetch(
        					AttributeContent("code")
        				),
        				EntityGroupFetch(
        					AttributeContent("code")
        				)
        			)
        		)
        	)
        )
	)
);